src/ui/src/builder/BuilderJournal.vue

```diff
@@ -72,6 +90,9 @@ const filters = ref<JournalFilters>({
 	instanceTypes: [],
 });
 
+const isDrawerOpen = ref(false);
+const selectedEntry = ref<JournalEntry | null>(null);
```

Comment:
1 . You can use `shallowRef` here to improve the perfs
2. I guess we can deduce `isDrawerOpen` 
```suggestion
const selectedEntry = shallowRef<JournalEntry | null>(null);
const isDrawerOpen = computed(() => !!selectedEntry.value);
```